.github/workflows/ci.yml

```diff
@@ -17,6 +17,8 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+      env:
+        - ACTIONS_ALLOW_UNSECURE_COMMANDS: True
```

Comment:
CI steps that use `actions/setup-python@v2` should work even without this env variable. I suppose that you got a CI error in your first commit because the branch didn't contain the latest CI fixes from master,